modules/vaos/app/controllers/vaos/v1/healthcare_services_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module VAOS
+  module V1
+    class HealthcareServicesController < VAOS::V1::BaseController
+      def index
+        response = fhir_service.search(:HealthcareService, query_string)
+        render json: response.body
+      end
+
+      private
+
+      def query_string
```

Comment:
Could this move into the `VAOS::V1::BaseController` I'm doing the same thing for the Organization resource 😄

Reply:
That's a great idea, should we wait for both PRs to be merged before refactoring?

Reply:
Looks like the query_string method was moved to the VAOS::V1::BaseController in this PR https://github.com/department-of-veterans-affairs/vets-api/pull/4294/files#diff-a460acb22659e8cecb99a052dce12588R24